src/main/java/org/apache/commons/lang3/BooleanUtils.java

```diff
@@ -681,13 +681,13 @@ public static Boolean toBooleanObject(final String str) {
             case 1: {
                 final char ch0 = str.charAt(0);
                 if (ch0 == 'y' || ch0 == 'Y' ||
-                    ch0 == 't' || ch0 == 'T' ||
-                    ch0 == '1') {
+                        ch0 == 't' || ch0 == 'T' ||
+                        ch0 == '1') {
```

Comment:
I think this is just cosmetic, not really fixing an issue. Changes like this make things harder to review. It's better to discuss with the developers of the project before changing code style.

Reply:
I agree with @kinow